### PR TITLE
[NEW] Add French Localizable

### PR DESCRIPTION
**Pull request by sowenjub**
@RocketChat/ios Add French Localizable

### CODE REVIEW

File: Rocket.Chat/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+"alert.connection.invalid_version.title" = "Oups!";
+"alert.connection.invalid_version.message" = "Le serveur auquel vous essayez de vous connecter utilise une version qui n'est plus supportée par l'app: %@.\n\nLa version %@ est requise.";
+
+"alert.settings.set_mail_app.title" = "Oups!";
+"alert.settings.set_mail_app.message" = "Aucun compte mail. Veuillez configurer un compte mail afin de pouvoir envoyer des emails.";
+
+"alert.feature.wip.title" = "Oups!";
+"alert.feature.wip.message" = "Cette fonctionnalité est en cours de développement, restez au courant!";
+
```
Comment (rafaelks) [inline Rocket.Chat/fr.lproj/Localizable.strings]: Could you please, add this new strings to the file (we have then in our English, if you want to look into it)?

```
"alert.socket_error.invalid_user.title" = "Session expired";
"alert.socket_error.invalid_user.message" = "Your session has expired. Please log in again.";

"alert.login_service_error.title" = "Authentication Error";
"alert.login_service_error.message" = "Could not authenticate using this service. Please, try again.";
```

**Comment by sowenjub** [comment]
@rafaelks Done, and I also added 2 new translations in prevision for https://github.com/RocketChat/Rocket.Chat.iOS/pull/849

**Comment by rafaelks** [comment]
@sowenjub You're fast! But added the strings to the English file, instead of French one. 🙈

**Comment by sowenjub** [comment]
@rafaelks 🤦‍♂️ Sorry about that, I tried to squeeze the patch between two meetings, that was a mistake!

**Comment by rafaelks** [comment]
Looking great! Thank you very much! :v:

**Comment by rafaelks** [comment]
@sowenjub Had to revert this... I didn't saw that, but your PR was pointing to `master`. Can you just open it pointing to `develop`, please?

**Pull request merged**